### DISCUSSION TITLE
Newbie question: I'm not able to reconstruct any object - can you help me

### DISCUSSION THREAD

**Discussion by msavazzi** [alicevision/Meshroom#1959] [Q&A]
Team,
sorry to bother, I'm very new to Meshroom and photogrammetry so it may sound dummy.
I've used a rotating plate and an SLR to take pictures of a small object I need to replicate as well as tried to take several free hand pictures with the iphone of the same object.
In both cases I do not get any reconstruction, the result is awfull. 
For sure is may fault :) so can anyone help me and give me some hints?

here the images
[ios part 1.zip](https://github.com/alicevision/Meshroom/files/11173041/ios.part.1.zip)

[ios part 2.zip](https://github.com/alicevision/Meshroom/files/11173046/ios.part.2.zip)

**Comment by msavazzi** [comment]
[ios part 3.zip](https://github.com/alicevision/Meshroom/files/11173059/ios.part.3.zip)
[SLR 1.zip](https://github.com/alicevision/Meshroom/files/11173062/SLR.1.zip)
[SLR 2.zip](https://github.com/alicevision/Meshroom/files/11173066/SLR.2.zip)

**Comment by msavazzi** [comment]
[SLR 3.zip](https://github.com/alicevision/Meshroom/files/11173068/SLR.3.zip)
[SLR 4.zip](https://github.com/alicevision/Meshroom/files/11173069/SLR.4.zip)

**Comment by msavazzi** [comment]
[SLR 5.zip](https://github.com/alicevision/Meshroom/files/11173072/SLR.5.zip)
[SLR 6.zip](https://github.com/alicevision/Meshroom/files/11173073/SLR.6.zip)

**Comment by Vertices365** [comment]
To be honest, I don't know what is wrong with the 2023 release. I was so hyped to try it out too and then suffered so many bugs an crashes that I returned to the good old 2021.1 release.
I attached a link to the 2021.1 release if you really need to get this done now and need something that works. I think sooner or later the community will have figured all the spookiness out and will fix it.
https://we.tl/t-Lf5ypDthRg
(I know it seems silly to just download stuff from a stranger on the internet but thats what I can offer you😅)
